apps/webClient/src/views/About/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/styles";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import FlareIcon from "@material-ui/icons/Flare";
+import Typography from "@material-ui/core/Typography";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Logo from "../../components/Logo/";
+import NavButton from "../../components/NavButton";
+
+interface Props {}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  contrastHeader: {
+    color: "white",
+    textTransform: "uppercase",
+  },
+}));
+
+const About: React.FC<Props> = ({}) => {
+  const classes = useStyles();
+
+  return (
+    <Box pt={16} height="100vh">
+      <Grid container justify="center">
+        <Link to="/">
+          <Logo />
+        </Link>
+      </Grid>
+      <Box mt={12}>
+        <Grid container>
+          <Grid
+            item
+            container
+            sm={2}
+            justify="flex-start"
+            alignItems="flex-start"
+          >
```

Comment:
I don't think this needs to be a `container` as it has no Grid item children.  It might be good to read through this page: https://material-ui.com/components/grid/ so that you can become the Grid boss and start teaching me things.